### DISCUSSION TITLE
Dramatically Different Image Quality Recently?

### DISCUSSION THREAD

**Discussion by throwingbones** [lllyasviel/Fooocus#1696] [Q&A] [answered]
(This software is fantastic. Thank you. This is not a complaint. I'm just not sure what changed, if anything. It's driving me crazy.)

Are you getting dramatically different image quality* using the exact same prompts and settings in the past week? I can't pinpoint when it happened, but it's recently. Sadly, I don't have before/after examples to post. I tried installing previous versions and running the same prompt to see if I could duplicate the effect, but I couldn't.

*For example, putting in a simple prompt "she is a vampire" used to often produce images of the same "person" with all of the default settings turned on. Now, it often does not. I could previously add to the prompt without the "person" changing. Things like, "she is a vampire holding a candle." Was something changed that reduced the consistency between images?

Also, "she is a vampire standing up" would often produce great images. With Fooocus Masterpiece turned on, the images were fantastic, including the faces. Now, the same prompt with the same settings produces images that are often not very good.

Without examples, this could all be my perception. I'm wondering if anyone else has noticed something similar or might have an explanation. Thank you.

**Comment by lllyasviel** [comment, marked as answer]
> I tried installing previous versions and running the same prompt to see if I could duplicate the effect, but I couldn't.

When you are using previous versions then you are using 100% previous versions. Previous versions are previous versions recorded by GitHub. 

>  I'm just not sure what changed

nope. Just tested 2.1.859 vs 851 vs 824 and several versions and results are all same

**Reply by throwingbones** [reply]
Thank you so much for the response, and for taking the time to test. Looking at the comment below, it may have just been the random seed vs. a specific seed that is giving me different results than I remember.

**Comment by usigg3** [comment]
Sounds like you may have run it with random seed turned off? I often do that when I want to explore what the words in the prompt do. I often get the same "person" then but with variations.

**Reply by throwingbones** [reply]
Thank you for this note. I've started recording the seed values for the characters I am using for my RPG sessions. This will help with the consistency. I generate images on the fly to visualize the scenes, so it is helpful if the characters look similar from image to image.
